internal/database/postgres/migration/000001_init-tables.up.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE users (
                        email VARCHAR UNIQUE NOT NULL,
                        password VARCHAR NOT NULL ,
                        phone VARCHAR,
-                       role VARCHAR,
+                       role VARCHAR NOT NULL ,
```

Comment:
in case those updates are already applied (and since they're in main, we should assume, that they're already applied, then this instruction won't run. For such changes, you need new migration, that alters table with new constraint